tests/test_operator.py

```diff
@@ -317,6 +317,29 @@ def test_sparsefunction_interp(self):
         # Exactly in the middle of 4 points, only 1 nonzero is 4
         assert sf1.data[0] == pytest.approx(1.0)
 
+    def test_sparsefunction_interp_dx(self):
+        """
+        Test interpolation of a SparseFunction from a Function
```

Comment:
shouldn't these tests (this and those above) go into test_interpolation ?

Reply:
Not sure ,lots of sparse  tests here too never know where to put what

Reply:
I'd say test_interpolation. Docstring may say more about why we want to test in particular interpolation from ``u.dx`` (instead of ``u``)